src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -196,6 +207,38 @@ describe('SamDebugConfigurationProvider', async () => {
                 assert.ok(!resources.includes(badResourceName))
             }
         })
+
+        it('Returns api function type resources as additional api configurations', async () => {
+            const bigYamlStr = `${makeSampleSamTemplateYaml(true)}
+            Events:
+                HelloWorld2:
+                    Type: Api
```

Comment:
Should `Api` (in the context of Cfn templates) also be a constant? Seeing it used in a couple places.

Reply:
It is, but this is the test yaml